chore: Adjust make command to support generating documentation for autogenerated resources (#3297)
mongodb/terraform-provider-mongodbatlas: docs/resources/custom_db_role_api.md

# Resource: mongodbatlas_custom_db_role_api

<!-- schema generated by tfplugindocs -->
## Schema

### Required

- `group_id` (String) Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.

> **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.

Reviewer: ```suggestion
**NOTE**: Groups and projects are synonymous terms. Your group ID is the same as your project ID. For existing groups, your group or project ID remains the same. The resource and corresponding endpoints use the term groups.
```

- `role_name` (String) Human-readable label that identifies the role for the request. This name must be unique for this custom role in this project.

### Optional

- `actions` (Attributes List) List of the individual privilege actions that the role grants. (see [below for nested schema](#nestedatt--actions))
- `inherited_roles` (Attributes Set) List of the built-in roles that this custom role inherits. (see [below for nested schema](#nestedatt--inherited_roles))

<a id="nestedatt--actions"></a>
### Nested Schema for `actions`

Required:

- `action` (String) Human-readable label that identifies the privilege action.

Optional:

> - `resources` (Attributes List) List of resources on which you grant the action. (see [below for nested schema](#nestedatt--actions--resources))

Reviewer: ```suggestion
- `resources` (Attributes List) List of resources on which you grant the action. (To lean more, [see nested schema](#nestedatt--actions--resources))
```

<a id="nestedatt--actions--resources"></a>
### Nested Schema for `actions.resources`

Required:

- `cluster` (Boolean) Flag that indicates whether to grant the action on the cluster resource. If `true`, MongoDB Cloud ignores the **actions.resources.collection** and **actions.resources.db** parameters.
- `collection` (String) Human-readable label that identifies the collection on which you grant the action to one MongoDB user. If you don't set this parameter, you grant the action to all collections in the database specified in the **actions.resources.db** parameter. If you set `"actions.resources.cluster" : true`, MongoDB Cloud ignores this parameter.
- `db` (String) Human-readable label that identifies the database on which you grant the action to one MongoDB user. If you set `"actions.resources.cluster" : true`, MongoDB Cloud ignores this parameter.

<a id="nestedatt--inherited_roles"></a>
### Nested Schema for `inherited_roles`

Required:

> - `db` (String) Human-readable label that identifies the database on which someone grants the action to one MongoDB user.

Reviewer: ```suggestion
- `db` (String) Human-readable label that identifies the database on which you grant the action to one MongoDB user.
```

> - `role` (String) Human-readable label that identifies the role inherited. Set this value to `admin` for every role except `read` or `readWrite`.

Reviewer: ```suggestion
- `role` (String) Human-readable label that identifies the inherited role. Set this value to `admin` for every role except `read` or `readWrite`.
```
